isotovideo

```diff
@@ -245,6 +246,8 @@ $bmwqemu::vars{TEST_GIT_HASH} = checkout_git_refspec($bmwqemu::vars{CASEDIR} =>
 # set a default distribution if the tests don't have one
 $testapi::distri = distribution->new;
 
+$bmwqemu::vars{WHEELS_DIR} ||= $bmwqemu::vars{PRODUCTDIR};
```

Comment:
I would still prefer to only fall back to CASEDIR. I assume wheels will mostly be used by lib/ or test modules in test distributions do not specific per product. So I see no sense in using product dir.

Reply:
That's why I left the thread unresolved 😄

Maybe @punkioudi you have some thoughts here as a future user? Where would you intuitively put the wheels.yaml?